skins/10h16/templates/Base_Layout.html.twig

```diff
@@ -16,13 +16,13 @@
     <meta property="og:site_name" content="Gemeinnnützige Wikimedia Fördergesellschaft mbH" />
     <meta property="og:description" content="Mir liegt Wikipedia am Herzen, deshalb habe ich gespendet. Denn was wäre das Netz ohne Wikipedia?" />
 
-    <link rel="shortcut icon" href="{$ basepath $}/skin/10h16/_assets/img/favicon.ico" />
-    <link rel="apple-touch-icon" href="{$ basepath $}/skin/10h16/_assets/img/apple-touch-icon.png" />
+    <link rel="shortcut icon" href="{$ basepath $}/skins/10h16/_assets/img/favicon.ico" />
+    <link rel="apple-touch-icon" href="{$ basepath $}/skins/10h16/_assets/img/apple-touch-icon.png" />
 
     {% block styles %}
     <link href="{$ basepath $}/res/css/fontcustom.css" rel="stylesheet" type="text/css" />
     <link href="{$ basepath $}/res/css/lib/jquery.ui/jquery-ui.css" rel="stylesheet" type="text/css" />
```

Comment:
Shouldn't we move those files, too? They also seem quite related to the skin.

Reply:
Oh, not part of the task.

Reply:
Well it is, but we don't need to do everything in one PR. Already did this stuff in https://github.com/wmde/FundraisingFrontend/pull/950 but that still needs a tiny bit of work